src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,201 @@
+
+```javascript
+  var listChild = (
+    <Item>
+      <ItemView
+        displays={React.Children.toArray(children)}
+      />
+    </Item>
+  );
+```
+
+## Creating Single Selectable Table version
+
+For `tableDisplay`, we start with this.
+
+```javascript
+  const tableDisplay = (
+    <Table
+      summaryId="standard-table"
+      summary="This table has standard padding"
+      numberOfColumns={mockData[0].cells.length}
+      dividerStyle="horizontal"
+      rowStyle="disclose"
+      bodyData={[
+        {
+          rows: createRows(mockData),
+        },
+      ]}
+      className={cx('table')}
+    />
+  );
+```
+
+`summaryId` is an element-id for the component, `summary` is just a brief text to describe the table's content, and `className` is just add some CSS. `numberOfColumns` is also pretty self-explanatory as how many columns you will need in the table. `dividerStyle` is used to put divisions between rows.
+
+`rowStyle` we define as 'disclose' because in our example our rows are selectable to then disclose further information upon selection.
+
+In our particular example, we created a mock data object we parse through to create the rows and cells. The `createRows` function which looks like this.
+
+```javascript
+const createRow = rowData => (
+  {
+    key: rowData.key,
+    isStriped: rowData.isStriped,
+    cells: createCellsForRow(rowData.cells),
+    discloseAction: {
+      metaData: { key: rowData.key },
+      discloseLabel: rowData.discloseText,
+      isDisclosed: selectedKey === rowData.key,
+      discloseCellIndex: 0,
+      onDisclose: handleRowDisclose,
+    },
+  }
+);
+```
+
+Each row needs a `key` which is a unique identifier, in our example it is 'row-0', 'row-1', etc. `isStriped` is set to 'true' every other row to give a close appearance to past item-collection.
+
+`discloseAction` is a special object for use when we select that row. `metaData` just contains a key for that row and `discloseLabel` is just a brief description the row such as 'Power Row 0 - Details'. `isDisclosed` is 'true' only when our component's state matches with the metadata key. `discloseCellIndex` is the index of the cell that provides the accessible link entry point.
+
+`onDisclose` is a simple state update function as seen here.
+
+```javascript
+const handleRowDisclose = (event, metaData) => {
+  event.preventDefault();
+  if (selectedKey !== metaData.key) {
+    setSelectedKey(metaData.key);
+  }
+};
+```
+
+For creating cells in each row we do this
+
+```javascript
+const createCellsForRow = cells => cells.map(cell => createCell(cell));
+const createCell = cell => ({ key: cell.key, children: cell.children, attrs: cell.attrs });
+```
+
+Cells are simple and just require a key, the children of said cell and then we use `attrs` to set `className` in our example like so.
+
+```javascript
+const attrs = { className: cx('content-display')};
+```
+
+## Creating a Cell Grid that looks like a Table
```

Comment:
How confident are we in the accessibility of the cell grid like table?

Reply:
No longer applicable since CellGrid example has been removed.